docs/Examples/DataAsJSONArray.md

```diff
@@ -0,0 +1,47 @@
+Send data as JSON array
```

Comment:
This example is great! 🎉 
How do you feel about changing the name of this example to something that is more verbose about the fact that we send an _Array_ instead of a _Dictionary_, which is the default one? I don't know if this one is that clear for beginners, which we aim for 😥